src/parser/shaman/elemental/modules/talents/Stormkeeper.js

```diff
@@ -19,9 +20,10 @@ class Stormkeeper extends Analyzer {
   constructor(...args) {
     super(...args);
     this.active = this.selectedCombatant.hasTalent(SPELLS.STORMKEEPER_TALENT.id);
+    this.addEventListener(Events.damage.by(SELECTED_PLAYER), this.onDamage);
```

Comment:
this one could have `AFFECTED_ABILITIES` as filter, the on_byPlayer_damage doesn't do anything special before checking them